components/salesforce_rest_api/README.md

```diff
@@ -2,6 +2,23 @@
 
 You can install the Pipedream Salesforce app in the [Accounts](https://pipedream.com/accounts) section of your account, or directly in a workflow.
```

Comment:
"you can connect your Salesforce account" is more accurate than "install the Pipedream Salesforce app", right? Or is that maybe that is more accurate in SF lingo?

Reply:
I think "you can connect your Salesforce account" sounds better since we're still on Pipedream vs. if we were to get Pipedream app listed within Salesforce's App Exchange Marketplace. Will fix this - thanks for the callout.